setup.cfg

```diff
@@ -1,6 +1,56 @@
-[bdist_wheel]
-universal=1
+[metadata]
+name = odltools
+summary = Tools for troubleshooting OpenDaylight
+description-file =
+    README.rst
```

Comment:
this is the long description in the setuptools, can we still have the short description and the long description?